src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -382,180 +379,171 @@ void GetZoneEquipmentData(EnergyPlusData &state)
         // for each equipment list
 
         CurrentModuleObject = "ZoneHVAC:EquipmentList";
+        auto &ip = state.dataInputProcessing->inputProcessor;
 
-        ZoneEquipListNum = state.dataInputProcessing->inputProcessor->getObjectItemNum(
-            state, CurrentModuleObject, state.dataZoneEquip->ZoneEquipConfig(ControlledZoneNum).EquipListName);
-        if (ZoneEquipListNum > 0) {
+        ZoneEquipListNum = ip->getObjectItemNum(state, CurrentModuleObject, state.dataZoneEquip->ZoneEquipConfig(ControlledZoneNum).EquipListName);
+
+        if (ZoneEquipListNum <= 0) {
+            ShowSevereError(state,
```

Comment:
`format`

Reply:
format all the things, done in fb44cc09c8ef7d82f3d61b9c373d5cdf42610385